debugging_protected_audience_api_services.md

```diff
@@ -132,6 +132,12 @@ PS_VERBOSITY = "10"
 The value of `CONSENTED_DEBUG_TOKEN` should match the debug token set in the
```

Comment:
The value of `CONSENTED_DEBUG_TOKEN` should match the debug token set in the
device. If set, the value must be a string of length 6 or more characters.

The behavior of `CONSENTED_DEBUG_TOKEN` is as following:
1. Setting to an empty string: This will turn consented debugging off for that server.
2. Setting to a string of length 6 or more characters: This causes consented debugging to be turned on, with the given value as the debug token for that server.
3. Setting to a non-empty string less than 6 characters will prevent the server from starting.